frontend/src/features/admin-form/settings/components/FormStatusToggle.tsx

```diff
@@ -49,7 +49,7 @@ export const FormStatusToggle = (): JSX.Element => {
 
     if (
       nextStatus === FormStatus.Public &&
-      responseMode === FormResponseMode.Encrypt
+      responseMode !== FormResponseMode.Email
```

Comment:
Better to check on `Encrypt | Multirespondent` which is how L69-70 decides render the controlled modal.